.github/workflows/release.yaml

```diff
@@ -0,0 +1,568 @@
+name: Concordium node release
+
+
+
+on:
+  push:
+    branches:
+      - lma/release/github_actions
+    tags:
+     - '*.*.*-*-rc'
+     - '*.*.*-*-alpha'
+
+env:
+  UBUNTU_VERSION: '20.04'
+  STATIC_LIBRARIES_IMAGE_TAG: 'rust-1.73.0_ghc-9.6.4'
+  RUST_VERSION: '1.73'
+  VERSION_TAG: '7.0.5-1-alpha'
+  STACK_VERSION: '2.13.1'
+  FLATBUFFERS_VERSION: '24.3.25'
```

Comment:
Bumped to most recent because https://github.com/google/flatbuffers/blob/master/CHANGELOG.md#23526-may-26-2023 gave issues with the make on the docker container on linux.